backends/qualcomm/CMakeLists.txt

```diff
@@ -253,7 +253,7 @@ target_link_libraries(qnn_executorch_backend
     qnn_executorch_header
     qnn_schema
     qnn_manager
-    executorch
+    executorch_no_prim_ops
```

Comment:
how do we link it with prim ops lib?

Reply:
In https://github.com/pytorch/executorch/blob/687425d1c17c99acdcee2c0617e84e93fc3e1c13/examples/qualcomm/CMakeLists.txt#L102-L108, maybe full_portable_ops_lib has it… Otherwise we need to add executorch explicitly

For other users, add executorch explicitly to binary helps

Reply:
TLDR is for users who use `qnn_executorch_backend` as a dependency, it should also have `executorch` as a dependency now.